### ISSUE TITLE
remove --extra option from trial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3374]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#3374|
|Type|task|
|Created|2008-07-30 00:32:27Z|
|Branch|https://github.com/twisted/twisted/tree/trial-extra-3374|

[#3372](https://github.com/twisted/twisted/issues/3372) has to be closed for a few releases (since 11.0) first.

<details><summary>Searchable metadata</summary>

```
trac-id__3374 3374
type__task task
reporter__glyph glyph
priority__lowest lowest
milestone__ 
branch__branches_trial_extra_3374 branches/trial-extra-3374
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1217377947000000 1217377947000000
changetime__1338135281000000 1338135281000000
version__None None
owner__thijs thijs
cc__cyli cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

This has been deprecated now.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [34317]) Branching to 'trial-extra-3374'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [34318]) remove deprecated --extra option for trial, add news file. refs [#3374](https://github.com/twisted/twisted/issues/3374)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

Up for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_8"></a>|@therve set owner to @thijstriemstra|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34461]) Merge trial-extra-3374: Remove the deprecated --extra option for trial, use the --testmodule option instead.

Author: thijs
Reviewer: therve
Fixes: [#3374](https://github.com/twisted/twisted/issues/3374)
